### PR TITLE
18123 rebased legal name FB with main

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#18123

*Description of changes:*
Rebasing again to keep feature branch up to date with main

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/enums/nameRequestStates.ts [hunk 1]
```diff
@@ -12,15 +12,13 @@ export enum NameRequestStates {
   INPROGRESS = 'INPROGRESS',
   REJECTED = 'REJECTED',
   NRO_UPDATING = 'NRO_UPDATING',
+  NO_CONTACT = 'NO_CONTACT',
+  INCORRECT_CONTACT = 'INCORRECT_CONTACT',
```
[the remark this answers is not in the document]
Comment (severinbeauvais) [reply src/enums/nameRequestStates.ts:16]: See how these are using in name-request-mixin.ts. I think they're used in the UI only (like NOT_FOUND).

**Review by severinbeauvais — COMMENTED** [review]
Wow, code changes back to October!

Basically looks good, though I found a couple things to update.

File: src/enums/nameRequestStates.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (JazzarKarim) [reply src/enums/nameRequestStates.ts]: Totally agree and even in `NameRequestErrorDialog`:
nrs states

Moved them to the block below.

File: src/resources/BreadCrumbResources.ts [hunk 2]
```diff
@@ -17,23 +21,26 @@ export function getEntityDashboardBreadcrumb (): BreadcrumbIF {
...
     href: `${sessionStorage.getItem('REGISTRY_HOME_URL')}dashboard/${getParams()}`
   }
 }
 
+/** Returns the breadcrumb to the My Business Registry page. */
 export function getMyBusinessRegistryBreadcrumb (): BreadcrumbIF {
   return {
     text: 'My Business Registry',
-    href: `${sessionStorage.getItem('BUSINESSES_URL')}business/${getParams()}`
+    href: `${sessionStorage.getItem('BUSINESSES_URL')}account/${store.getAccountId}/business`
```
[the remark this answers is not in the document]
Comment (JazzarKarim) [reply src/resources/BreadCrumbResources.ts:36]: Got it. Line 10 is now: `const accountId = store.getAccountId || 0`.

**Review by severinbeauvais — APPROVED** [review]

**Review by argush3 — APPROVED** [review]

**Pull request merged**